compiler/circle-execution-plan/src/ExecutionPlanner.cpp

```diff
@@ -345,7 +554,7 @@ void ExecutionPlanner::create_alloc_node_inform_vector(bool null_consts, bool nu
   for (size_t i = 0; i < _ordered_nodes.size(); i++)
   {
     auto circle_node = loco::must_cast<luci::CircleNode *>(_ordered_nodes[i]);
-    auto node_size = 1;
+    auto node_size = circle_node->rank() > 0 ? 1 : 0;
```

Comment:
Are you sure?
node with rank == 0 is a scalar

Reply:
done, please see https://github.com/Samsung/ONE/pull/9893